docs/source/admin_guide/breaking-upgrade.md

```diff
@@ -0,0 +1,160 @@
+# Upgrade - Redeployment for Breaking Changes
+
+For versions that are known to require redeployment when upgrading, the steps for upgrade include:
+
+- Back up user data by following [this guide](./backup.md).
```

Comment:
It is recommended to use numbered lists for steps that must be performed in a given order 
```suggestion
1. Back up user data by following [the backup guide](./backup.md) in the documentation.
```